week-5/README.md

```diff
@@ -0,0 +1,45 @@
+# AJAX
+
+## Client/Server architecture
+
+A **server** is device or program that provides functionality to other programs or devices. There are database servers, mail servers, game servers, etc. The vast majority of these servers are accessed over the internet.
+
+The server communicates with **clients**. Clients can be a web browser, a Slack app, your phone, etc.
+
+Client–server systems use the **request–response** model: a client sends a request to the server, which performs some action and sends a response back to the client, typically with a result or acknowledgement.
+
```

Comment:
This is an important concept to pin early. What is client/server, what is frontend/backend etc...? The students, in the last class, seemed confused by that for way longer than they should have.

A suggestion is to even incorporate this game we do in the taster classes: https://codeyourfuture.github.io/taster-class-master/intro/internet-intro.html .. explaining how internet works, and just emphasising what's a client and a server